### ISSUE TITLE
Enums with numeric values defined in enumValues are not generated correctly

### ISSUE THREAD

**Issue by BigsonLvrocha** [dotansimha/graphql-code-generator#3137]
**Describe the bug**
 

There are two, enum given numeric values in enumValues config options are generated as string and value `0` will not be converted at all

**To Reproduce**

give numeric values to map enum fields in enumValues config option

1. My GraphQL schema:

```graphql
enum FooEnum {
  FOO
  BAR
}
```

2. My GraphQL operations:

no problems with operation, just with generated type, graphql will convert correctly

3. My `codegen.yml` config file:

```yaml
overwrite: true
schema: './data/schema.graphql'
documents: null
generates:
  packages/api/src/generated/schema.ts:
    plugins:
      - add: '/* istanbul ignore file */'
      - typescript
      - typescript-resolvers
    config:
      namingConvention:
        enumValues: keep
      enumValues:
        FooEnum:
          FOO: 1
          BAR: 0
```

**Expected behavior**

The generated code should be
```typescript
export enum FooEnum {
  FOO = 1
  BAR = 0
}
```
but it is 
```typescript
export enum FooEnum {
  FOO = '1'
  BAR = 'BAR'
}
```

https://codesandbox.io/s/graphql-codegen-issue-template-1lmxr

**Environment:**

- OS: Linux
- `@graphql-codegen/...`: 
```json
{
    "@graphql-codegen/add": "^1.8.3",
    "@graphql-codegen/cli": "^1.9.1",
    "@graphql-codegen/introspection": "^1.9.1",
    "@graphql-codegen/typescript": "^1.9.1",
    "@graphql-codegen/typescript-resolvers": "^1.9.1",
}
```
- NodeJS: 12.13.0

**Additional context**

graphql will convert correctly because I have the enum resolver defined explicitly as

```typescript
export const resolvers = {
  FooEnum: {
    FOO: 1,
    BAR: 0,
  },
};
```

**Comment by dotansimha** [comment]
@BigsonLvrocha I think I managed to fix that in: https://github.com/dotansimha/graphql-code-generator/pull/3187 

Can you please try `1.9.2-alpha-4be25b2b.82`?

**Comment by dotansimha** [comment]
Available in `v1.10.0`.

**Comment by BigsonLvrocha** [comment]
I was on a trip, got the computer now, it's fixed, thanks :)
